extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -649,37 +649,98 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
   });
 
   describe('#unlayoutCallback', () => {
+    beforeEach(() => {
+      const setup = createImplTag({
+        width: '300',
+        height: '150',
+      }, element, impl, env);
+      element = setup[0];
+      impl = setup[1];
+      env = setup[2];
+      impl.buildCallback();
+      impl.win.ampAdSlotIdCounter = 1;
+      expect(impl.element.getAttribute('data-amp-slot-index')).to.not.be.ok;
+      impl.layoutMeasureExecuted_ = true;
+      impl.uiHandler = {applyUnlayoutUI: () => {}};
+      const placeholder = doc.createElement('div');
```

Comment:
nit: use more descriptive name than placeholder, e.g. adSlotPlaceholder  or something like that